shoreditch.php

```diff
@@ -28,6 +28,20 @@ function shoreditch_civicrm_xmlMenu(&$files) {
  */
 function shoreditch_civicrm_install() {
   _shoreditch_civix_civicrm_install();
+
+  // Unset "custom css url" (customCSSURL) field
+  $customCSSUrl = Civi::settings()->get("customCSSURL");
+  if (strpos($customCSSUrl, 'org.civicrm.shoreditch') !== FALSE) {
+    civicrm_api3('setting', 'create', [
+      'customCSSURL' => ''
+    ]);
+  }
+
+  // Set "shoreditch" as the backend theme.
+  civicrm_api3('setting', 'create', [
+    'theme_frontend' => 'default',
+    'theme_backend' => 'shoreditch'
+  ]);
 }
```

Comment:
@vinuvarshith to avoid the duplication of this setup code, I suggest this:

- Inside `CRM_Shoreditch_Upgrader`, create a `setShoreditchAsBackendTheme()` method and then move all this code there
- Change `upgrade_0001` to simply call this new method instead
- Add an `install()` method to `CRM_Shoreditch_Upgrader` (this will be automatically called during the installation) and call the new method from it as well.

Reply:
done